src/plugin/reflection_class.cc

```diff
@@ -13,6 +17,21 @@ namespace chapel {
   void ReflectionClassGenerator::Generate(Printer* printer) {
     WriteIntroduction(printer);
 
+    string module_name;
+    if (!package_name.empty()) {
+      module_name = package_name;
+    } else {
+      module_name = default_file_name;
+    }
+
+    printer->Print("module $module_name$ {\n", "module_name", module_name);
+    printer->Print("\n");
+    printer->Indent();
+
+    printer->Print(
+      "use Encoding;\n"
```

Comment:
Minor nit: the spacing in this call is a bit off

Reply:
Are you pointing towards the spacing in the `Print` call? Somehow it is looking fine to me :sweat_smile: 

Ohh! Or you are talking about the extra line -> no 30?

Reply:
I'm talking about how the "use Encoding" string is on a separate line from the start of the call when it seems like it would fit on one line (like line 27 does).  It also seems like it would be better to do a separate call for the second `"\n"` (to maintain the readability you are going for) or put it in same string (to avoid spreading the contents of the file too much)

Reply:
Ohh! Got it. I will make the changes.